psydac/api/feec.py

```diff
@@ -86,51 +86,88 @@ def __init__(self, mapping, *spaces):
     #--------------------------------------------------------------------------
     @property
     def dim(self):
+        """dimension of the physical and logical space"""
         return self._dim
 
     @property
     def V0(self):
+        """First space of the De Rham sequence : H1 space"""
         return self._spaces[0]
 
     @property
     def V1(self):
+        """Second space of the De Rham sequence :
+        1d : L2 space
+        2d : either Hdiv or Hcurl space
+        3d : Hcurl space"""
         return self._spaces[1]
 
     @property
     def V2(self):
+        """Third space of the De Rham sequence :
+        2d : L2 space
+        3d : Hdiv space"""
         return self._spaces[2]
 
     @property
     def V3(self):
+        """Fourth space of the De Rham sequence : L2 space in 3d"""
         return self._spaces[3]
 
     @property
     def H1vec(self):
+        """Vectorial H1 space built has cartesian product of V0"""
         assert self.has_vec
         return self._H1vec
 
     @property
     def spaces(self):
+        """Spaces of the proper de Rham sequence (excluding Hvec)"""
         return self._spaces
 
     @property
     def mapping(self):
+        """The mapping from the logical space to the physical space of the discrete De Rham."""
         return self._mapping
 
     @property
     def callable_mapping(self):
+        """The mapping as a callable"""
         return self._callable_mapping
 
     @property
     def derivatives_as_matrices(self):
+        """Differential operators of the De Rham sequence as BlockLinearOperator"""
```

Comment:
`as BlockLinearOperator objects`, and you can specify on which type of objects these operators act

@yguclu is it important that they are of type `BlockLinearOperator`, or just specifying `as LinearOperator` is good enough?

also probably the name `derivatives_as_matrices` is no longer appropriate ?

Reply:
Good point! These objects are not always of type `BlockLinearOperator`; in the 1D case you get a `DirectionalDerivativeOperator` instance. Hence we should write `as LinearOperator objects` as you suggested.

The name is not perfect, but these are the differential operators appearing in the de Rham sequence of the coefficient spaces. We usually refer to them as "matrices" in our publications, and I do not have a better name... 😅

Reply:
Ok ! Just asking :)

Reply:
(regarding the objects on which they act, I guess we do not need to be very specific -- reminding that they act on `Vectors` is probably good enough)

Reply:
so I suggest: 
```
"""Differential operators of the De Rham sequence as LinearOperator (acting on coefficient vectors)"""
```

Reply:
Done in [49ed53e](https://github.com/pyccel/psydac/pull/349/commits/49ed53eec0a2ac1e5117de0e818720027b2a6e36)